src/Collections-Sequenceable/OrderedCollection.class.st

```diff
@@ -685,6 +685,23 @@ OrderedCollection >> removeAt: index [
 	^removed
 ]
 
+{ #category : #removing }
+OrderedCollection >> removeDuplicates [
```

Comment:
This method should be moved to `SequenceableCollection` so it becomes available for any of these collections.

Reply:
If this is done, we need to do specific implementation in case of non-growable collections.

Reply:
I just moved this function from somewhere else in the image to get this PR working.

Further improvements should be done in a new PR (But I agree with you)